File: app/src/main/java/me/miran/mhdstuff/activity/settings/SettingsActivity.java
package me.miran.mhdstuff.activity.settings;

import android.os.Bundle;
import android.widget.Button;

import androidx.annotation.Nullable;

import me.miran.mhdstuff.R;
import me.miran.mhdstuff.activity.base.BaseActivity;

public class SettingsActivity extends BaseActivity {


    public SettingsActivity() {
        super(R.string.settings);
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);


        setContentView(R.layout.activity_settings);

        Button lineTest = findViewById(R.id.open_dev_settings);

        lineTest.setOnClickListener(view -> {
            startActivity(DevSettingsActivity.class);
        });
    }
}

File: app/src/main/java/me/miran/mhdstuff/activity/settings/SettingsActivity.kt
package me.miran.mhdstuff.activity.settings

import androidx.compose.runtime.Composable
import me.miran.mhdstuff.activity.base.NavigationActivity
import me.miran.mhdstuff.R

class SettingsActivity : NavigationActivity(R.string.settings) {

    @Composable
    override fun CreateNavigation() {
        NavigationItem(R.drawable.code, "dev options", DevSettingsActivity::class)
    }


}